libraries/rush-lib/src/logic/TempProjectHelper.ts

```diff
@@ -20,6 +20,7 @@ export class TempProjectHelper {
    * Deletes the existing tarball and creates a tarball for the given rush project
    */
   public createTempProjectTarball(rushProject: RushConfigurationProject): void {
+    // FIXME: ignore for now, since WorkspaceInstallManager takes responsible for installing for split workspace packages, which is not related with this helper
```

Comment:
Maybe we should enforce that `useWorkspace=true` in **rush.json** if there is a split workspace

Reply:
Added a validation in `RushConfiguration`: packageManager must be "pnpm" and "useWorkspaces" must be enabled.